scripts/install-chef-suse.sh

```diff
@@ -344,7 +344,21 @@ EOF
     exit 1
 fi
 
-reset_crowbar
+if ! (systemctl -q is-active crowbar-init.service && reset_crowbar ); then
```

Comment:
is that after you finished the installation?

Reply:
crowbar-init is a "shim" service that is stopped after crowbar has been spun up properly (it only runs during the "bootstrap" phase or the "upgrade")

Reply:
My vague memory is that it was introduced because the main app uses ActiveRecord which requires a db connection, whereas we needed a service which could run before the db was available in order to bootstrap it (or possibly upgrade it, not sure).  But I might have remembered this wrong.  @rsalevsky can confirm or deny.

Reply:
Seems like another way to do this could be to test -f $crowbar_install_dir/crowbar-install-failed the way the case above tests for success.